### PR TITLE
fix: fixes #218

### LINKED ISSUE #218: Data Overflow Beyond Card UI
When **three or more topics** are added under the slug, they extend beyond the **profile user interface**
Screenshot from 2023-02-08 11-10-27
# To overcome this problem we can :
- **Limit the topics to atmost 3**
- **Or can make changes in the UI to scale any number of topics**

### PR DESCRIPTION
**Pull request by ArkaPrabhaChowdhury**
I have also made sure that DSA and DS remain capitalized. 
image

### CODE REVIEW

File: data/users/vasujhawar2001.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     },
     {
       "slug": "programming",
-      "topics": ["Javascript", "Data Structures", "Java"]
+      "topics": ["Javascript", "DS", "Java"]
```
Comment (codewithvoid) [inline data/users/vasujhawar2001.json:12]: @ArkaPrabhaChowdhury let's change all to DSA

File: src/pages/index.js [hunk 2]
```diff
@@ -49,6 +49,9 @@ export default function Home({ users, categories }) {
     }
 
     function titleCase(str) {
+      if(str=="DSA" || str=="DS"){
```
Comment (codewithvoid) [inline src/pages/index.js:52]: let's create a `Set` of topics which will be ignored from title casing and check here if topic string is part of that list.

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by ArkaPrabhaChowdhury** [comment]
@codewithvoid PTAL.

File: src/pages/index.js [hunk 3]
```diff
@@ -49,6 +49,9 @@ export default function Home({ users, categories }) {
     }
 
     function titleCase(str) {
+      if(capitalize.has(str)){
```
Comment (codewithvoid) [inline src/pages/index.js:52]: nit: `capitalize` -> `capitalizedTopics`

**Review by codewithvoid — APPROVED** [review]

File: src/pages/index.js (hunk 3 above)
Comment (codewithvoid) [reply src/pages/index.js:52]: @ArkaPrabhaChowdhury this is ready to submit. Can you make this change.

Comment (ArkaPrabhaChowdhury) [reply src/pages/index.js:52]: @codewithvoid Sure!

**Comment by ArkaPrabhaChowdhury** [comment]
@codewithvoid PTAL.

**Pull request merged**